client/my-sites/plugins/style.scss

```diff
@@ -523,6 +523,32 @@ body.is-section-plugins .plugin-management-wrapper {
 		padding-left: 2rem;
 		padding-right: 2rem;
 	}
+
+	.dataviews-wrapper {
+		.dataviews__view-actions {
+			display: none;
+		}
+		.dataviews-view-table {
+			border: 1px solid var(--color-neutral-5);
+			border-radius: 2px;
+	
+			.dataviews-view-table__row {
+				img, .dataviews-view-table__row svg {
+					max-width: 35px;
+					margin-right: 15px;
+				}
+	
+				.dataviews-view-table-selection-checkbox{
+					padding-left: 0;
+				}
+			}
+	
+			.dataviews-view-table__checkbox-column {
+				padding-left: 18px;
+			}
+		}
+	}
```

Comment:
Not sure if we should do that. I know we should produce minimal UI mismatch, but this directly contradicts what we discussed in our last meeting: introducing more UI drifting from the official UI library.

Reply:
Yep, I agree with you on that. We need to reuse what A4A did, even though their code is encapsulated in a module. Before submitting it for review, I'll try to reuse it or propose refactoring it in another PR. For now, it was a test to ensure the DataViews can deliver a similar interface.

Thanks for the early review, @zaguiini!